TLM/TLM/Manager/Impl/LaneConnectionManager.cs

```diff
@@ -71,7 +71,7 @@ public class LaneConnectionManager
         private bool IsHeadingTowardsStartNode(uint sourceLaneId) {
             NetLane[] laneBuffer = NetManager.instance.m_lanes.m_buffer;
             ushort segmentId = laneBuffer[sourceLaneId].m_segment;
-            NetSegment segment = GetSeg(segmentId);
+            NetSegment segment = segmentId.ToSegment();
```

Comment:
ref ref :D

Reply:
you should take a look at macsurgeys code! he refuses to use ref as a matter of principle!

Reply:
I was just kidding!
@DaEgi01 In any case if you didn't notice I already fixed it.

Reply:
![image](https://user-images.githubusercontent.com/26344691/143261936-621de55e-af7b-42ec-ac95-23e6fec8a451.png)

Reply:
lol :D